complete/src/main/java/hello/Application.java

```diff
@@ -2,8 +2,14 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
 @SpringBootApplication
+@EnableJpaRepositories(basePackageClasses=hello.repositories.UsersRepository.class)
```

Comment:
These JPA annotations are redundant. If you want to discuss them in the text, fine, but it seems distracting, given that the guide is supposed to be about MySQL.